src/test/java/org/minions/devfund/TaskOneTest.java

```diff
@@ -0,0 +1,24 @@
+package org.minions.devfund;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TaskOneTest {
+
+    /**
+     * Verify some TaskOne method.
+     */
+    @Test
+    public void testOne() {
+        TaskOne taskOne = new TaskOne();
+        int result = taskOne.countChar("fizzbuzz",'z');
+        Assert.assertEquals(4, result);
```

Comment:
Consider using a static import for Assert

Reply:
what do you mean ?